event_model/__init__.py

```diff
@@ -1822,3 +1822,42 @@ def default(self, obj):
                 return obj.item()
             return obj.tolist()
         return json.JSONEncoder.default(self, obj)
+
+
+def replay(gen, callback, burst=False):
+    """
+    Emit documents to a callback with realistic time spacing.
+
+    Parameters
+    ----------
+    gen: iterable
+        Expected to yield (name, doc) pairs
+    callback: callable
+        Expected signature: callback(name, doc)
+    burst: bool, optional
+        If True, emit as fast as possible, ignoring timing. False by default.
+    """
+    DOCUMENTS_WITHOUT_A_TIME = {'datum', 'datum_page', 'resource'}
```

Comment:
Looks like a list.  Why the curly braces?

Reply:
Micro-optimization to use a set instead of a list.  It make the init a bit slower, makes `x in DOCUMENTS_WITHOUT_A_TIME` a bit faster.   Removes the `[n]` access because there are no meaningful semantics in the order of these values.